src/xmpp/iq.c

```diff
@@ -1612,35 +1612,54 @@ _version_get_handler(xmpp_stanza_t* const stanza)
     xmpp_ctx_t* const ctx = connection_get_ctx();
     const char* id = xmpp_stanza_get_id(stanza);
     const char* from = xmpp_stanza_get_from(stanza);
+    ProfAccount* account = accounts_get_account(session_get_account_name());
+    GString* client = account->client != NULL ? g_string_new(account->client) : NULL;
```

Comment:
Maybe I missed that part in the code, but if you don't need the `GString` functionality, there's no need to create a `GString`. Instead you could simply use `account->client`.

Reply:
I just needed to copy this part in memory because I didn't want to make any changes in the original of `account->client`.

Reply:
```suggestion
    auto_char char* client = account->client != NULL ? strdup(account->client) : NULL;
```

We're moving away from using `GString` since its overhead is really not required in most cases. This is one of those were a simply `strdup()` is sufficient. If you use `auto_char` the compiler will take care of `free()`'ing that newly allocated string.